acc_test/import_resource_test.go

```diff
@@ -233,6 +239,20 @@ func TestKafkaUserResource_importBasic(t *testing.T) {
 					"wait_for_state",
 				},
 			},
+			{
+				PreConfig: func() {
+					fmt.Println("Sleep for 6 minutes to wait for Kafka cluster to be ready for resize.")
```

Comment:
This feels like it could create flakey tests? Why is wait_for_state == "RUNNING" insufficient?

Reply:
The new CDC resize API will need to check the cluster state, which is based on the metrics. For Kafka clusters, around 5 minutes of metrics will be checked to determine if the cluster is healthy enough for the resize. So after reach running, need to wait for around 5 minutes for the metrics being collected.